File: problemas/iniciante/2702/2702.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let entrada = input.trim().split("\n")
    let [Ca, Ba, Pa] = entrada.shift().trim().split(' ').map(Number);
    let [Cr, Br, Pr] = entrada.shift().trim().split(' ').map(Number);

    let resultado = 0
    resultado += (Cr > Ca) ? (Cr - Ca) : 0;
    resultado += (Br > Ba) ? (Br - Ba) : 0;
    resultado += (Pr > Pa) ? (Pr - Pa) : 0;

    console.log(resultado);
});

File: problemas/iniciante/2702/2702.py
def main():
    Ca, Ba, Pa = map(int, input().split())
    Cr, Br, Pr = map(int, input().split())

    saida = 0
    saida += (Cr - Ca) if Cr > Ca else 0
    saida += (Br - Ba) if Br > Ba else 0
    saida += (Pr - Pa) if Pr > Pa else 0
    
    print(saida)

if __name__ == "__main__":
    main()
